README.md

```diff
@@ -28,11 +28,13 @@ Below is an example of regexp patterns used for autolinking, modified in the `co
             "links": [
                 {
                     "Pattern": "(LHS)",
-                    "Template": "[LHS](https://docs.mattermost.com/process/training.html#lhs)"
+                    "Template": "[LHS](https://docs.mattermost.com/process/training.html#lhs)",
+                    "Scope": ["team1/off-topic"]
                 },
                 {
                     "Pattern": "(RHS)",
-                    "Template": "[RHS](https://docs.mattermost.com/process/training.html#rhs)"
+                    "Template": "[RHS](https://docs.mattermost.com/process/training.html#rhs)",
+                    "Scope": ["team2", "team3", "team4/town-square"]
```

Comment:
I think I put this here while thinking about what I wanted, without adding a note to implement it.

Reply:
actually, i don't like it.  removing.